integration-tests/chopsticks/src/network/index.ts

```diff
@@ -0,0 +1,40 @@
+import * as SpiritnetConfig from './spiritnet.js'
+import * as PolkadotConfig from './relay.js'
+import * as HydraDxConfig from './hydraDx.js'
+import { ChainConfigs } from './types.js'
+
+/*
+ * Get an environment variable and throw an error if it is not set.
+ */
+function getEnvVariable(name: string): string {
```

Comment:
Then I would rename this function to:

```suggestion
function getRequiredEnvVariable(name: string): string {
```